src/Network/GRPC/MQTT/Message/Request.hs

```diff
@@ -0,0 +1,259 @@
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE ImplicitPrelude #-}
```

Comment:
This hasn't been addressed. Is there a reason you are migrating things away from `Relude`?

Reply:
I haven't been using `relude` because I find it more difficult to use over `prelude`. Ideally we'd be able to include `relude` by overriding the prelude module in cabal mixins which would also enable us to opt in or out of aspects of `relude`, but then proto generated Haskell files wont compile. Theres experimental extensions to cabal that would let us get around this by splitting the proto Haskell files into a separate private package without a mixin stanza, but I don't think depending on an experimental feature is worth it.

Unless I'm mistaken, I think `relude` primary justification is that it's more convenient to use? I don't really find it helps me in practice though (many modules actually require several additional imports to use `relude` compared with just using `prelude`), so I been avoiding it in the newer modules. Some other frustrations I have with it:
 * `Show (show)` is not exported. Importing `Text.Show` then clashes with `relude`'s version of `show` which will infer `Show` implementations in ways that drops formatting I want and vice versa.
 * All haddock identifier linking is redirected to the documentation package's `relude` depends on (e.g. unliftio). 
 * `grpc-mqtt` makes very light usage of `IO` transformer stacks so instantiating all of the unlifted `IO` functions `relude` re-exports seems redundant.

Reply:
A strong argument for `relude` is the much simpler text/string/bytestring conversion tools, which I use often. I'm in favor of keeping `relude`.